MACSio.spec

```diff
@@ -0,0 +1,68 @@
+Name:           MACSio
+Version:        1.1
+Release:        1%{?commit:.git%{shortcommit}}%{?dist}
+Summary:        A Multi-purpose, Application-Centric, Scalable I/O Proxy Application
+
+License:        GPL
+URL:            https://github.com/LLNL/MACSio
+Source0:        https://github.com/LLNL/%{name}/archive/v%{version}.tar.gz
+
+BuildRequires:  cmake
+BuildRequires:  json-cwx
+BuildRequires:  hdf5-mpich
```

Comment:
This should be required by the `hdf5-mpich-devel` below, so not really needed here.
```suggestion
```

Reply:
Removed.